### PR TITLE
Add UseResponse

### PR DESCRIPTION
**Pull request by DanilaAka**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``

- [x] Your additions are ordered alphabetically.
- [ ] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [ ] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please check PR guidelines and check the boxes.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: non-free.md [hunk 1]
```diff
@@ -108,7 +108,7 @@
   
 ## Ticketing
   * [SupportPal](https://www.supportpal.com/) `⊘ Proprietary` - Powerful help desk software - easy, fast and intuitive. ([Demo](http://demo.supportpal.com/)) `PHP`
-
+  *[UseResponse] (https://www.useresponse.com/)`⊘ Proprietary` - All-in-one solution (feedback community portal, help desk, live chat, knowledge base).([Demo](https://www.useresponse.com/trial))`PHP`
```
Comment (nodiscc) [inline non-free.md]: * Extra space after `[`
* Missing space before ``⊘ Proprietary`
* Missing space before ``PHP`
 * Missing space after dot

**Comment by Kickball** [comment]
@DanilaAka this PR will be closed in 7 days unless you provide an update.

**Comment by UseResponse** [comment]
Updated According to nodiscc corrections

**Comment by UseResponse** [comment]
Updated

**Comment by UseResponse** [comment]
Updated

File: non-free.md (hunk 1 above)
Comment (UseResponse) [inline non-free.md]: updated

Comment (nodiscc) [inline non-free.md]: I don't see any update

**Comment by Kickball** [comment]
@UseResponse I'm not seeing your updates anywhere. Are you sure you were pushing to the correct repository/branch?

**Comment by DanilaAka** [comment]
Hello, Edward!
I'm not a big professional in this sphere. Could you send me the link to the clear tutorial of how to do that?

На 5 авг. 2017 г., 12:34, в 12:34, Edward Dickson <notifications@github.com> написал:>@UseResponse I'm not seeing your updates anywhere. Are you sure you

**Pull request closed without merging**